web/client/utils/__tests__/GeoStoryUtils-test.js

```diff
@@ -355,8 +355,11 @@ describe("GeoStory Utils", () => {
         };
         const res = createMapObject(DEFAULT_MAP_OPTIONS, {
             mapOptions: {
+                scrollWheelZoom: false,
                 interactions: {
-                    mouseClick: false
+                    mouseWheelZoom: false,
+                    mouseClick: false,
+                    dragPan: true
                 }
```

Comment:
I would avoid to change the test structure.
Here it seems that the expectation is to merge these value so probably we should apply the replacement just for the layer key.

We will need also an additional test for the layers key